### ISSUE TITLE
Could not pass event PlayerUseUnknownEntityEvent

### ISSUE THREAD

**Issue by xyfwdy** [FancyMcPlugins/FancyNpcs#41]
FancyNpcs v2.0.2.1

This server is running Folia version git-Folia-"f6e91e9" (MC: 1.20.1) (Implementing API version 1.20.1-R0.1-SNAPSHOT) (Git: f6e91e9)
You are running the latest version
```
Could not pass event PlayerUseUnknownEntityEvent to FancyNpcs v2.0.2.1
java.lang.NoSuchMethodError: 'org.bukkit.util.Vector com.destroystokyo.paper.event.player.PlayerUseUnknownEntityEvent.getClickedRelativePosition()'
	at de.oliver.fancynpcs.listeners.PlayerUseUnknownEntityListener.onPlayerUseUnknownEntity(PlayerUseUnknownEntityListener.java:18) ~[FancyNpcs-2.0.2.1.jar:?]
	at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor368.execute(Unknown Source) ~[?:?]
	at org.bukkit.plugin.EventExecutor$2.execute(EventExecutor.java:77) ~[folia-api-1.20.1-R0.1-SNAPSHOT.jar:?]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:77) ~[folia-api-1.20.1-R0.1-SNAPSHOT.jar:git-Folia-"f6e91e9"]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[folia-api-1.20.1-R0.1-SNAPSHOT.jar:?]
	at io.papermc.paper.plugin.manager.PaperEventManager.callEvent(PaperEventManager.java:54) ~[folia-1.20.1.jar:git-Folia-"f6e91e9"]
	at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.callEvent(PaperPluginManagerImpl.java:126) ~[folia-1.20.1.jar:git-Folia-"f6e91e9"]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:615) ~[folia-api-1.20.1-R0.1-SNAPSHOT.jar:?]
	at net.minecraft.server.network.ServerGamePacketListenerImpl.callPlayerUseUnknownEntityEvent(ServerGamePacketListenerImpl.java:2998) ~[?:?]
	at net.minecraft.server.network.ServerGamePacketListenerImpl$5.a(ServerGamePacketListenerImpl.java:2980) ~[?:?]
	at net.minecraft.network.protocol.game.ServerboundInteractPacket$InteractionAction.dispatch(ServerboundInteractPacket.java:131) ~[?:?]
	at net.minecraft.network.protocol.game.ServerboundInteractPacket.dispatch(ServerboundInteractPacket.java:80) ~[?:?]
	at net.minecraft.server.network.ServerGamePacketListenerImpl.handleInteract(ServerGamePacketListenerImpl.java:2977) ~[?:?]
	at net.minecraft.network.protocol.game.ServerboundInteractPacket.handle(ServerboundInteractPacket.java:67) ~[?:?]
	at net.minecraft.network.protocol.game.ServerboundInteractPacket.handle(ServerboundInteractPacket.java:12) ~[?:?]
	at net.minecraft.network.protocol.PacketUtils.lambda$ensureRunningOnSameThread$0(PacketUtils.java:52) ~[?:?]
	at net.minecraft.network.protocol.PacketUtils.lambda$ensureRunningOnSameThread$1(PacketUtils.java:75) ~[?:?]
	at io.papermc.paper.threadedregions.EntityScheduler.executeTick(EntityScheduler.java:173) ~[folia-1.20.1.jar:git-Folia-"f6e91e9"]
	at net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:1524) ~[folia-1.20.1.jar:git-Folia-"f6e91e9"]
	at io.papermc.paper.threadedregions.TickRegions$ConcreteRegionTickHandle.tickRegion(TickRegions.java:360) ~[folia-1.20.1.jar:git-Folia-"f6e91e9"]
	at io.papermc.paper.threadedregions.TickRegionScheduler$RegionScheduleHandle.runTick(TickRegionScheduler.java:385) ~[folia-1.20.1.jar:git-Folia-"f6e91e9"]
	at ca.spottedleaf.concurrentutil.scheduler.SchedulerThreadPool$TickThreadRunner.run(SchedulerThreadPool.java:525) ~[folia-1.20.1.jar:git-Folia-"f6e91e9"]
	at java.lang.Thread.run(Thread.java:1623) ~[?:?]
```

**Comment by OliverSchlueter** [comment]
Yes, you need to wait for Folia to merge the latest paper patches, then it should work

**Comment by xii69** [comment]
> Yes, you need to wait for Folia to merge the latest paper patches, then it should work

Same problem with Paper 1.19.4

**Comment by OliverSchlueter** [comment]
> > Yes, you need to wait for Folia to merge the latest paper patches, then it should work
> 
> Same problem with Paper 1.19.4

Fixed for 1.19.4, see 86674d3ffec16e5d037fbec233867ddab009a2db 
Just waiting for Folia to merge paper patches now
